### DISCUSSION TITLE
security connection review

### DISCUSSION THREAD

**Discussion by lowerclass** [GorvGoyl/ChatGPT-Writer#6] [Report Bugs and Issues]
I am having trouble passing the Cloudflare test. I did visit [chat.openai.com] to pass the Cloudflare security check, and came back to Gmail, but the disclaimer is still on, and the text can not be generated.

**Comment by GorvGoyl** [comment]
it seems to be working fine for me.. could you close the chatgpt writer and open it again and try?

**Reply by GorvGoyl** [reply]
for the brave browser, go to brave://settings/shields and disable `Prevent sites from fingerprinting me based on my language preferences`


![screenshot 2023-01-03 at 19 06 03@2x](https://user-images.githubusercontent.com/7106086/210354629-4bca77d4-28b5-4eb0-ab50-78429aa4f368.png)

**Comment by lowerclass** [comment]
Will try. Thank you!
________________________________

**Comment by lowerclass** [comment]
Thank you! This worked.

út 3. 1. 2023 v 13:09 odesílatel Gourav Goyal ***@***.***>
napsal:

**Comment by WillemRoman** [comment]
This is also happening for me in Chrome.. Any solution for Cloudflare test error in Chrome?

**Reply by engalaapt2023** [reply]
Same in Chrome ..
